memory-pair-game/card.js

```diff
@@ -0,0 +1,46 @@
+export default class Card {
+  static render({ value, onClick }, parentElement) {
+    const element = document.createElement('figure')
+    element.className = 'card'
+
+    element.innerHTML = `<img src="images/${value}.jpg" alt="@${value}">`
+    element.addEventListener('click', onClick)
```

Comment:
Lets say that you app has grew up and it has 12x12 playing field. Do you realy want to create 144 event listeners? And cause GC to remove 144 event listeners each time you're removing cards from the DOM?
If you do want, then consider to clear all references to each card upon deleting them from DOM(maybe you already did, I'm not sure) or removeEventListeners explicitly.
I would have use event delegation. Maybe in this case doing direct event listener is ok, but in big aplications, its very easy do forget smth and get a memory leak because of that.

Reply:
Actually, I do. Each `Game#start()` creates new array with cards, so all links are cleared. However, you are right, event delegation may be better solution, I will think about it. Thank you!